packages/Input/__tests__/Input.spec.jsx

```diff
@@ -315,6 +315,17 @@ describe('Input', () => {
         'Reveal additional information about Some field.'
       )
     })
+
+    it('allows custom tooltip names to be used', () => {
+      const Tool = Tooltip
+
+      const input = mount(<Input label="Some field" tooltip={<Tool>The tooltip</Tool>} />)
+
+      expect(input.find(StandaloneIcon)).toHaveProp(
+        'a11yText',
+        'Reveal additional information about Some field.'
+      )
+    })
```

Comment:
I am pretty skeptical of the value and readability test...

It seems like it is testing the PropType validation of Tooltip with a pretty obscure usage of renaming the Tooltip component. Also, the assertion that is being made on the StandaloneIcon doesn't match the description of the test: "allows custom tooltip names to be used".

I would recommend to just not have this test at all since it seems to be testing airbnb-prop-types and not our own code.